src/detection.h

```diff
@@ -9,11 +10,10 @@
 void tim1_up_isr(void);
 
 /* State machine status */
-#define EMIT_UNDEFINED 0
-#define EMIT_ON 1
-#define EMIT_ON_ADC_ON 2
-#define EMIT_OFF 3
-#define EMIT_OFF_ADC_ON 4
+#define A 1
+#define B 2
+#define C 3
+#define D 4
```

Comment:
I'd rather use a descriptive name or, if that is not possible, just numbers (i.e.: case 0, case 1...). At least this way we do not need the defines... :wink:

Reply:
Done :)